docs/reference/ilm/set-up-lifecycle-policy.asciidoc

```diff
@@ -1,6 +1,6 @@
 [role="xpack"]
 [[set-up-lifecycle-policy]]
-== Configure a lifecycle policy [[ilm-policy-definition]]
+== Configure a lifecycle policy [[ilm-configure-policy]]
```

Comment:
> == Configure a lifecycle policy [[ilm-configure-policy]]

From the explanation in the PR, I'm not sure what the goal of this change is.  If your intention is to change https://www.elastic.co/guide/en/elasticsearch/reference/master/ilm-policy-definition.html, the source file is actually https://raw.githubusercontent.com/elastic/elasticsearch/master/docs/reference/redirects.asciidoc

Reply:
I'm actually surprised it didn't cause an error as-is, since the anchor IDs are supposed to be unique.  It makes me think it's not even recognized. My recommendation would be to remove that anchor rather than change it.